src/core/io/mpiio/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_library(mpiio SHARED mpiio.cpp)
 target_include_directories(mpiio PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries(mpiio PRIVATE EspressoConfig EspressoCore MPI::MPI_CXX)
+install(TARGETS mpiio LIBRARY DESTINATION ${PYTHON_INSTDIR}/espressomd)
```

Comment:
What is the purpose of the new `install()` line here and in the other submodules?

Reply:
These libraries were missing from the installation and hence made the espressomd module unusable.